dmsite/frontend/src/ClassificationPage/ClassificationPage.jsx

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react';
+import {
+  TabContent, TabPane, Nav, NavItem, NavLink, Button, Row, Col, Input, Table, ListGroup, ListGroupItem
+} from 'reactstrap';
+import classnames from 'classnames';
+import './ClassificationInfo.css'
+
+let FETCH_URL = 'http://localhost:8000/classify_files';
+//TODO: all of this needs style changes
+
+class ClassificationPage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      activeTab: 0,
+      results: [],
+    };
+  }
+
+  toggle(tab) {
+    console.log("switching to " + tab);
```

Comment:
use console.debug for easier filtering of messages

Reply:
Oops, left that in there by accident and forgot to remove it. Thanks for noticing!